sdcm/sla/libs/sla_utils.py

```diff
@@ -375,3 +386,45 @@ def clean_auth(entities_list_of_dict):
                         auth.drop()
                     except Exception as error:  # pylint: disable=broad-except
                         LOGGER.error("Failed to drop '%s'. Error: %s", auth.name, error)
+
+    @staticmethod
+    def scylla_shares_per_scheduler_group(node, scheduler_group_name: str = None) -> dict:
+        """
+        output example:
+        ['scylla_scheduler_shares{group="sl:default",shard="0"} 1000.000000',
+         'scylla_scheduler_shares{group="streaming",shard="0"} 200.000000'
+        ]
+        """
+        scheduler_regex = re.compile(r".*group=\"(?P<group>.*)\",.* (?P<shares>\d+)")
```

Comment:
sheduler_regex and part for matching should be part of `scylla_scheduler_shares` - this could return dict.
This method then would just get desired group from this dict

Reply:
done